modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -1161,7 +1161,7 @@ mod tests {
             }
         );
         assert_eq!(
-            recover_scan_idxes_for_activity(TweakIdx(10), &BTreeSet::new(), |cur_idx| async move {
+            recover_scan_idxes_for_activity(TweakIdx(9), &BTreeSet::new(), |cur_idx| async move {
```

Comment:
The fact that the argument passed to this function, which acts somewhat like a start of a range is not inclusive is very weird, IMO. :D

Reply:
Isn't it inclusive? Maybe I missed something, I just mechanically moved the `.prev()` call out of the `recover_scan_idxes_for_activity` fn into the callers.